src/ngraph_mark_for_clustering.cc

```diff
@@ -528,6 +530,7 @@ Status MarkForClustering(Graph* graph) {
       if (!confirmation_constraint_ok) {
         NGRAPH_VLOG(5) << "Node does not meet confirmation constraints: "
                        << node->name();
+        no_support_histogram[node->type_string()]++;
```

Comment:
`no_support_histogram` smushes together "op absolutely not supported" and "supported under certain conditions"

would it be possible to separate them in `ConfirmationOk` into 2 separate histograms.

That is an extra headache, u may have to split up `ConfirmationOk`

I suggest using `NGraphPlacementRequested` for one check (which is dummy right now) and `ConfirmationOk` for the other

Something like this in pseudo/python code
`NGraphPlacementRequested  = lambda op : op.name in dictionary`
`ConfirmationOk  = lambda op : dictionary[op.name](op)`

Reply:
As we discussed, since `NGraphPlacementRequested` doesn't take confirmation_function_map as an input, instead use a global map to keep track the node that not supported.